src/components/Navbar/Drawer/index.tsx

```diff
@@ -41,6 +42,7 @@ interface Props {
   hideCloseButton?: boolean;
   children: ReactNode;
   closeOnNavigation?: boolean;
+  closeOnBlur?: boolean;
```

Comment:
nit: we need a better naming for `closeOnBlur` as it's not clear what it stands for. Maybe `shouldCloseDrawer`?